logstash-core/src/test/java/org/logstash/ackedqueue/QueueTest.java

```diff
@@ -174,9 +175,8 @@ public void multiWriteSamePage() throws IOException {
     @Test
     public void writeMultiPage() throws IOException {
         List<Queueable> elements = Arrays.asList(new StringElement("foobarbaz1"), new StringElement("foobarbaz2"), new StringElement("foobarbaz3"), new StringElement("foobarbaz4"));
-        int singleElementCapacity = singleElementCapacityForByteBufferPageIO(elements.get(0));
         try (Queue q = new Queue(
-            TestSettings.volatileQueueSettings(2 * singleElementCapacity))) {
+            TestSettings.volatileQueueSettings(multipleElementsCapacityForByteBufferPageIO(elements.get(0), 2)))) {
```

Comment:
This function name is delightfully long. No change required; just noting that I chuckled a bit when noticing the length ;)

Future possibly just rename both of these methods (multiple and single) to `computeCapacityForByteBufferPageIO(element, count)`  and use `1` for count when using the single case. (again , no change required)

Reply:
+1

Reply:
renamed